app/internal/page/certificate_provider_address.go

```diff
@@ -64,7 +65,16 @@
 				addresses, err := addressClient.LookupPostcode(r.Context(), data.Form.LookupPostcode)
 				if err != nil {
 					logger.Print(err)
-					data.Errors.Add("lookup-postcode", validation.CustomError{Label: "couldNotLookupPostcode"})
+
+					if errors.As(err, &place.BadRequestError{}) {
+						data.Errors.Add("lookup-postcode", validation.EnterError{Label: "invalidPostcode"})
+					} else {
+						data.Errors.Add("lookup-postcode", validation.CustomError{Label: "couldNotLookupPostcode"})
+					}
+				} else if len(addresses) == 0 {
+					logger.Print("No addresses found for " + data.Form.LookupPostcode)
```

Comment:
I'd say we don't need these logs since we won't take any action from them

Reply:
@hawx I've wondered about how our logging works for a while - is there any kind of auto-escaping/sanitizing going on behind the scene in the logging package we use or will this be raw input?

In this case we're pretty sure its at least a valid form postcode otherwise we'd get a 400 response but still feels like we should be cautious with user input in logs.

Reply:
Good point - logging probably isn't the best place for this but I'd still be interested in how often this happens. Maybe something to look at with RUM if we decide to track it.